### ISSUE TITLE
Support for Angular 12

### ISSUE THREAD

**Issue by shyamal890** [IKatsuba/ngx-ssr#433]
Its not working with Angular 12 giving dependency error
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: smarttask-info@0.0.0
npm ERR! Found: @angular/common@12.2.8
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~12.2.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^11.0.6" from @ngx-ssr/cache@1.1.1
npm ERR! node_modules/@ngx-ssr/cache
npm ERR!   @ngx-ssr/cache@"*" from the root project
```

**Comment by IKatsuba** [comment]
@shyamal890, thank you for your report! I am working on it :)

**Comment by shanmugapriyak1788** [comment]
@IKatsuba Can you provide us with the end date for this? We have a production application with this utility. Our angular upgrade 13 has failed since due to this dependency. We planning to roll out the upgrade in the next 3 days. Please let us know whether it is possible within the stipulated deadline.

**Comment by IKatsuba** [comment]
> @IKatsuba Can you provide us with the end date for this? We have a production application with this utility. Our angular upgrade 13 has failed since due to this dependency. We planning to roll out the upgrade in the next 3 days. Please let us know whether it is possible within the stipulated deadline.
> 
> 

Oh, sorry guys. I will try to do it tonight

**Comment by IKatsuba** [comment]
@shanmugapriyak1788, please, try beta versions of the packages
```
npm i @ngx-ssr/{package-name}@beta
```

**Comment by IKatsuba** [comment]
:tada: This issue is included in version ngx-ssr-cache@1.2.0-beta.1 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by shanmugapriyak1788** [comment]
Thank you so  much @IKatsuba

**Comment by exequiel09** [comment]
ViewEngine is removed in Angular. See: https://blog.angular.io/angular-v13-is-now-available-cce66f7bc296#5b7f

We need to release a new version compatible with v13 and above @IKatsuba 

![Screen Shot 2022-10-13 at 8 15 57 AM](https://user-images.githubusercontent.com/1289472/195470610-7151ef84-073b-4fa1-a652-9a34543c4305.png)
